sgrep.py

```diff
@@ -450,11 +462,11 @@ def load_config(location: Optional[str] = None) -> Any:
             else:
                 print_error_exit(f"{loc} is not a file or folder!")
         else:
-            print_error_exit(f"unable to find a config file in {base_path.resolve()}")
```

Comment:
@DrewDennison I feel like we should still exit here, they passed a config and we didn't find it